### PR TITLE
Add instructions on how to run locally

### PR DESCRIPTION
**Pull request by tmeasday**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,33 @@ To facilitate upgrading in the future, removing and adding features, this is the
...
+### Building and running locally
+
+1. Ensure all dependencies are installed with `yarn install`
+2. Build + watch the code locally: `yarn dev`
+3. Run a build of all the CLI's stories against a Chromatic project: `yarn chromatic -t <token>`.
+
+#### Running against staging
+
+```bash
+CHROMATIC_INDEX_URL=https://www.staging-chromatic.com/graphql yarn chromatic -t 253df72b53d2
```
Comment (ghengeveld) [inline README.md:47]:
```suggestion
CHROMATIC_INDEX_URL=https://index.staging-chromatic.com yarn chromatic -t 253df72b53d2
```

File: README.md [hunk 2]
```diff
@@ -35,6 +35,33 @@ To facilitate upgrading in the future, removing and adding features, this is the
...
+```
+
+#### Running against development
+
+To test against a local development version of the Chromatic stack, use
+
+```
+CHROMATIC_INDEX_URL=https://www.dev-chromatic.com/graphql yarn chromatic -t <token>
```
Comment (ghengeveld) [inline README.md:55]:
```suggestion
CHROMATIC_INDEX_URL=https://index.dev-chromatic.com yarn chromatic -t <token>
```

**Review by ghengeveld — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tmeasday) [reply README.md:47]: Hmmmm.. I'm pretty sure you need the `/graphql` bit. Plus it is not really different (as www proxies to the index), but I find it simpler to just make it www.

PS I tested these commands they definitely work!

Comment (ghengeveld) [reply README.md:47]: I always run with `CHROMATIC_INDEX_URL=https://index.staging-chromatic.com` without the path 🤷‍♂️
Seems more sensible to me to set `INDEX_URL` to the actual index URL.

Comment (tmeasday) [reply README.md]: TIL

**Pull request merged**